index.html

```diff
@@ -75,10 +80,10 @@ <h3 class="about-conn">Let's Connect</h3>
     </div>
     <section class="works">
       <ul class="works-card-list">
-        <li class="works-card">
+        <li class="works-card first-card">
```

Comment:
- [ ] Good job designing the project cards 👍 but it would be better if you reduce the width of your project cards so that they can match that of the Figma template.
- [ ] Kindly apply this to all your project cards.
- Your design:
![Annotation 2022-02-26 102126](https://user-images.githubusercontent.com/85113567/155837922-082ba8ea-67db-4d61-a644-d21e54a44f91.jpg)
- Figma design:
![Annotation 2022-02-26 102212](https://user-images.githubusercontent.com/85113567/155837928-52489d50-dfa4-4ef4-857a-294095812442.jpg)

Reply:
* I have fixed the issues in the heading.

Reply:
* Fixed issue basing on figma design.